src/web/src/components/Posts/PostInfo.tsx

```diff
@@ -5,20 +5,19 @@ import AdminButtons from '../AdminButtons';
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
-      margin: '0',
+      marginLeft: '2rem',
       padding: '0',
       display: 'flex',
+      borderLeft: '1.5px solid #999999',
       flexDirection: 'column',
       justifyContent: 'center',
-      borderLeft: '2.5px solid #707070',
-      [theme.breakpoints.up('lg')]: {
-        width: '22rem',
-      },
```

Comment:
Re-add the breakpoint to utilize the whitespace on larger screens. 

https://github.com/Seneca-CDOT/telescope/pull/2026#issuecomment-808944743